WordPress/Classes/ViewRelated/Comments/CommentModerationBar.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+class CommentModerationBar: UIView {
+
+    // MARK: - Properties
+
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var pendingButton: UIButton!
+
+    // MARK: - Init
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+
+        guard let view = loadViewFromNib() else {
+            DDLogError("CommentModerationBar: Failed loading view from nib.")
+            return
+        }
+
+        view.frame = self.bounds
+        configureView()
+        self.addSubview(view)
+    }
+
+}
+
+// MARK: - Private Extension
+
+private extension CommentModerationBar {
+
+    // MARK: - Configure
+
+    func loadViewFromNib() -> UIView? {
+        let nib = UINib(nibName: "\(CommentModerationBar.self)", bundle: Bundle.main)
+        return nib.instantiate(withOwner: self, options: nil).first as? UIView
+    }
+
+    func configureView() {
+        configureBackground()
+        pendingButton.configureFor(.pending)
+    }
+
+    func configureBackground() {
+        contentView.backgroundColor = .tertiaryFill
+        contentView.layer.cornerRadius = 15
```

Comment:
I'll move all the style related setting to `WPStyleGuide+CommentDetail` after I finish the bar (i.e. when I get all the styles worked out).